addon/index.js

```diff
@@ -359,6 +361,15 @@ function installExperiment(experiment) {
   }
 }
 
+function uninstallSelf() {
+  // First, kick out all the experiment add-ons
+  Object.keys(store.installedAddons).forEach(id => {
+    uninstallExperiment({addon_id: id});
+  });
```

Comment:
Is `uninstall-all` actually used anywhere? Actually meant to remove that event listener, because I can't find anything in the frontend that calls that.